scripts/publish.sh

```diff
@@ -14,6 +15,9 @@ check() {
     fi
 }
 
+# delete all files required for native testing
+delete_native_tests
```

Comment:
So I guess before the .unitypackage would include all of our tests because Unity will pull in everything from the Assets/Plugins folder right?